alacritty/src/config/window.rs

```diff
@@ -52,6 +49,10 @@ pub struct WindowConfig {
     /// Background opacity from 0.0 to 1.0.
     pub opacity: Percentage,
 
+    // XXX This wants SerdeReplace...
+    #[cfg(target_os = "macos")]
+    pub option_as_alt: OptionAsAlt,
+
```

Comment:
@chrisduerr, Could you help to resolve this one? The struct wants `SerdeReplace`. The type has all sorts of traits implemented already including serde's serialize/deserialize... Right now what I see is to wrap, but I don't want to wrap....

Reply:
The thing is that `SerdeReplace` needs to be implemented for this enum to work, so we get the `replace` function.

Since we do not own it, we obviously can't use `ConfigDeserialize`, so what you should do is add it to `impl_replace` in `alacritty_config/src/lib.rs`. Should be pretty simple, but probably easier for you with access to a mac.

Reply:
I'm just not happy that other crates started depending on winit.

Reply:
You mean `alacritty_config`? It also depends on log that's just kinda the nature of the crate.